engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/UnwrapValidatedValueTest.java

```diff
@@ -118,87 +113,6 @@ public void shouldRaiseExceptionIfNoMatchingUnwrapperIsFound() {
 		validator.validate( new Order() );
 	}
 
-	@Test
```

Comment:
Ok, but shouldn't we have a test for specifying the option (via the API) on the constraint level? Or are there other tests for that?